zeroconf.py

```diff
@@ -1617,20 +1614,20 @@ def find(cls, zc=None, timeout=5, interfaces=InterfaceChoice.All):
         return tuple(sorted(listener.found_services))
 
 
-def get_all_addresses(address_family):
+def get_all_addresses():
     return list(set(
-        addr['addr']
-        for iface in netifaces.interfaces()
-        for addr in netifaces.ifaddresses(iface).get(address_family, [])
-        if addr.get('netmask') != HOST_ONLY_NETWORK_MASK
+        addr.ip
+        for iface in ifaddr.get_adapters()
+        for addr in iface.ips
+        if type(addr.ip) is str and len(addr.ip.split(".")) == 4
```

Comment:
This change removes host-only interface skipping introduced in b5e9e944e6f3c990862b3b03831bb988579ed340, that check needs to stay (with modifications, naturally).